app/controllers/two_factor_authentication/otp_verification_controller.rb

```diff
@@ -4,7 +4,10 @@ class OtpVerificationController < DeviseController
 
     def show
       assign_variables_for_otp_verification_show_view
-      analytics.track_event(Analytics::MULTI_FACTOR_AUTH_ENTER_OTP_VISIT, analytics_properties)
+      analytics.track_event(
+        Analytics::MULTI_FACTOR_AUTH_ENTER_OTP_VISIT,
+        analytics_properties.except(:confirmation_for_phone_change)
```

Comment:
On second thought, you are right. I thought we wouldn't be interested in that property for a page visit, but I think we might, and it doesn't hurt to have extra properties. I'll revert this.